python/ccxt/test/test_async.py

```diff
@@ -161,7 +161,14 @@ def exception_message(exc):
     return message
 
 
-def exit_script(code=0):
+def compare_exception_type(exc, exceptionType):
```

Comment:
@ttodua I would suggest removing this method it adds complexity for no reason, I would rather see the instance of directly inside the catch

Reply:
done ;